src/utils/cellSets/getContainingCellSetsProperties.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Returns the properties of the cellSets that contains the cellId
+ * (by the order specified in hierarchy children)
+ * @param {int} cellId The id of the cell we are searching for.
+ * @param {array} cellSetClassKeys The cellSetClass we are searching inside of
```

Comment:
Btw, why do you set this parameter as an array? Why not just a string? Is there a case where we want to specify more than one `cellSetClassKey`?

Reply:
Yeah in the embedding we want to display all the cluster names that have the cellId - it can be both scratchpad and louvain